.puppet-lint.rc

```diff
@@ -0,0 +1 @@
+--relative
```

Comment:
can you explain why you changed this? It should not be required to run the tests.

Reply:
When I tried to git commit, puppet-lint was executed in pre-commit and it failed.
Without this fix, I couldn't git commit, so I added it.
```
~/jkkitakita/puppet-telegraf master +1 ❯ git commit -m "hoge"
Running pre-commit hooks
Check for trailing whitespace....................[TrailingWhitespace] OK
Runs puppet-lint on modified files only..................[PuppetLint] FAILED
Errors on lines you didn't modify:

jkkitakita/puppet-telegraf/manifests/params.pp:5:7:ERROR: telegraf::params not in autoload module layout (autoloader_layout)

✗ One or more pre-commit hooks failed
```